src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -392,19 +392,21 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     pass_config->enable<ngraph::pass::ConvertDetectionOutput1ToDetectionOutput8>();
 
     if (useLpt) {
+        const auto defaultPrecisions = ngraph::pass::low_precision::LowPrecision::getDefaultPrecisions(nGraphFunc);
```

Comment:
Default precisions depend on FakeQuantize levels. Please, identify default precisions in one place upper and reuse it. Don't pass `getDefaultPrecisions` result. Default precisions identification you can find below:
```
        // for GNA networks reference execution
        bool updatePrecision = true;
        bool hasINT16orINT32Levels = ngraph::pass::low_precision::LowPrecision::isFQLevelsPresent(
                nGraphFunc,
                {levels::int16, levels::int16_narrow_range,
                 levels::int32, levels::int32_narrow_range});
        if (hasINT16orINT32Levels) {
            updatePrecision = false;
            LowPrecision::setDefaultPrecisions(precision_set::int8_int16_int32_support);

            supportedPrecisions = std::vector<OperationPrecisionRestriction>({});
        }
```

Reply:
done, thank you.